e2e/app/page/notebook-cell.ts

```diff
@@ -0,0 +1,152 @@
+import {ElementHandle, Frame, Page} from 'puppeteer';
+import {getPropValue} from 'utils/element-utils';
+
+export enum CellType {
+  // To append to css selector
+  Code = '.code_cell',
+  Markdown = '.text_cell',
+  Any = '',
+}
+
+export default class NotebookCell {
+
+  private readonly page: Page;
+  private iframe: Frame; // Jupyter notebook in iframe
+  private cellIndex: number;
+  private readonly cellType: CellType;
+
+
+  constructor(page: Page, cellType: CellType, cellIndex?: number) {
+    this.page = page;
+    this.cellIndex = cellIndex;
```

Comment:
cellIndex should be documented somewhere. For example, it's not clear to me whether this index includes both code and output cells.